### ISSUE TITLE
Test Suite Installation Error

### ISSUE THREAD

**Issue by elma16** [firedrakeproject/firedrake#1784]
Hi,

I am very new to Firedrake, and have recently installed Firedrake on my Mac (version 10.15.6). Running the demo scripts seems to work perfectly fine, but when I run the test suite as recommended, I get the following error.

```
tests/test_0init.py ...                                                  [  0%]
tests/test_tsfc_interface.py ..........                                  [  0%]
tests/demos/test_demos_run.py .......ss.......s....                      [  0%]
tests/equation_bcs/test_bcs_reconstruct.py .                             [  0%]
tests/equation_bcs/test_equation_bcs.py ......make: *** [test] Error 147
```

Does this mean that something is wrong with my installation? If so, how can I fix it?

**Comment by dham** [comment]
There are some issues with numpy on Mac right now that might be the cause of this. I'd recommend just going ahead and using Firedrake. If what you are doing is affected by the bug then you'll get a crash and we'll try to do something for that particular case. You won't silently get the wrong answer due to this bug. @JDBetteridge is working on an actual fix.

**Comment by jfregin** [comment]
Receiving this very same error now on an openSUSE install as well. error occurs later during the tests though. 
error line says `test/multigrid/test_embedded_transfer.py make: *** [Makefile:65: test] Error 147`.

**Comment by wence-** [comment]
Should have been fixed by the merge of #1885. If you fetch a new `firedrake-install` script and install afresh, things should work.
